Dockerfile

```diff
@@ -1,5 +1,7 @@
 # Build node feature discovery
-FROM golang:1.8 as builder
+FROM BASEIMAGE1 as builder
+
+CROSS_BUILD_COPY qemu-QEMUARCH-static /usr/bin/
```

Comment:
Could this be commented out, so that no mangling would be needed for amd64. In addition, you can pass `QEMUARCH` as a build arg